components/price.js

```diff
@@ -43,39 +43,37 @@ export function PriceProvider ({ price, children }) {
   )
 }
 
+const STORAGE_KEY = 'asSats'
+
+const carousel = [
+  'fiat',
+  'yep',
+  '1btc',
+  'blockHeight',
+  'chainFee',
+  'halving'
+]
+
```

Comment:
Introduced a carousel concept here as better reflects the idea of clicking through items in the header.